src/swagger-specs/streaming-ingestion.yml

```diff
@@ -0,0 +1,638 @@
+openapi: 3.0.1
+info:
+  title: Streaming Ingestion API
+  description: |-
+    Using this endpoint you can bring your data into Adobe Experience
+    Platform through streaming ingestion. Streaming ingestion
+    lets users send data to Platform in real time from client and server-side devices.
```

Comment:
We can avoid "using" becoming a gerund here by revising to: "Streaming ingestion allows you to send data from client and server-side devices to Experience Platform in real time. Platform supports the use of data inlets to stream incoming experience data, which is persisted in streaming-enabled datasets within the data lake. Data inlets can be configured to automatically authenticate the data they collect, ensuring that the data is coming from a trusted source."